packages/core/src/SkipMenu/SkipMenu.stories.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { SkipMenu } from '.'
+import { Flex } from '..'
+
+const skipLinks = [
+  {
+    label: 'Skip to main content',
+    targetId: 'main-content',
+  },
+  {
+    label: 'Skip to listings',
+    targetId: 'listings-section',
+  },
+  {
+    label: 'Skip to filters',
+    targetId: 'filter-section',
+  },
+  {
+    label: 'Skip to footer',
+    targetId: 'footer',
+  },
+]
+
+storiesOf('SkipMenu', module).add('default', () => (
```

Comment:
Mind changing this to CSF?

Reply:
Yeap. Just FYI this is the template that you get when you run `rush create`, so maybe that needs to be updated.